src/mixins/audit.mixin.ts

```diff
@@ -59,6 +59,7 @@ export function AuditRepositoryMixin<
         entityId: entity.getId(),
         actedOn: this.entityClass.modelName,
         actionKey: opts.actionKey,
+        tenantId: user.tenantId,
```

Comment:
what if the user.tenantId is undefined 
then defaultTenantId should be saved

Reply:
user.tenantId can never be undefined. as it is mentioned as required in our authentication service. it user didn't provide its values then he/she will not able to login.
there is also one defaultTenantId, but if tenantid can not be NULL then why do we need to mention defaultid here